scrapers/sis_scraper/util.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+# Python standard library
+from copy import deepcopy
+import datetime
+import time
+
+# "COMPUTER SCIENCE I" => "Computer Science I"
+def normalize_class_name(name):
+    name = list(name)
+
+    for i in range(1, len(name)):
+        if name[i - 1] == " ":
+            continue
+        name[i] = name[i].lower()
+    return "".join(name)
```

Comment:
Is there a reason why this function can't be replaced with `name.title()` or `string.capitalize(title)`?

Reply:
TIL string.title() is a thing, good catch

Reply:
updated https://github.com/quacs/quacs/pull/690/commits/906eaf6f9c0b03dce98335f22513e25a3fadce6e